examples/pytorch/rgcn/experimental/entity_classify_dist.py

```diff
@@ -274,29 +315,32 @@ def sample_blocks(self, seeds):
         norms = []
         ntypes = []
         seeds = th.LongTensor(np.asarray(seeds))
-        cur = seeds
+        gpb = self.g.get_partition_book()
+        cur = gpb.map_to_homo_nid(seeds, 'paper')
         for fanout in self.fanouts:
-            frontier = self.sample_neighbors(self.g, cur, fanout, replace=True)
-            etypes = self.g.edata[dgl.ETYPE][frontier.edata[dgl.EID]]
-            norm = self.g.edata['norm'][frontier.edata[dgl.EID]]
+            frontier = self.sample_neighbors(self.g, cur, fanout, replace=False)
```

Comment:
yes, the mini-batch is stored as homogeneous graphs.

Reply:
This is a discrepancy and makes the code hard to understand, especially since `DistEmbedLayer` and `NeighborSampler` are initialized with the same `g`.  Do we explain the mechanic somewhere (e.g. where are the conversions from/to heterogeneous graph taking place)?

Reply:
this is a temporary solution. we need to fix heterograph mini-batch training first before we can fix this problem.
We'll have to use a doc to explain what is happening here.